harvester_e2e_tests/integrations/test_upgrade.py

```diff
@@ -240,22 +239,6 @@ def check(self, data):
             for func in self.intercepts():
                 func(data)
 
-        def intercept_v121_vm(self, data):
-            if "v1.2.1" != api_client.cluster_version.raw:
-                return
-            if self._v121_vm:
```

Comment:
Perhaps also drop class variable `_v121_vm` (Line 232) since it's only referenced by `intercept_v121_vm`?